product_quick_bom/tests/test_quick_bom.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2015 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class TestQuickBom(TransactionCase):
+
+    def setUp(self):
+        super(TestQuickBom, self).setUp()
+        self.computer = self.env.ref(
+            'product.product_product_5_product_template')
+        self.ram = self.env.ref('product.product_product_13_product_template')
+        self.hard_drive = self.env.ref(
+            'product.product_product_17_product_template')
+        self.cpu = self.env.ref('product.product_product_22_product_template')
+
+    def test_create_bom(self):
+
+        computer = self.computer
```

Comment:
you do not need to set computer = self.computer
you can directly use self.computer